### PR TITLE
Release/1.9.2

### PR DESCRIPTION
**Pull request by thoma202**
Fixed links 
Fixed anonymous function 
Fixed UI / UX 
Fixed categories pagination
Fixed no profile country choosing

### CODE REVIEW

File: classes/EbayCountrySpec.php [hunk 1]
```diff
@@ -287,7 +377,8 @@ public function checkCountry()
 	**/
 	private function _getCountry()
 	{
-		$ebay_profile = EbayProfile::getCurrent();		
+		$ebay_profile = EbayProfile::getCurrent();	
+		if($ebay_profile)
```
Comment (Quetzacoalt91) [inline classes/EbayCountrySpec.php]: What happen if this test return false ? Notices will be thrown.

File: de.php [hunk 2]
```diff
@@ -28,10 +26,9 @@
...
 $_MODULE['<{ebay}prestashop>ebay_1761efb946aa43c14af989665dae691a'] = 'Produkt (e)';
-$_MODULE['<{ebay}prestashop>ebay_3defc5eecc87a01b7ff5dfc4fcd8898a'] = 'Siehe unsere Bewertungen';
-$_MODULE['<{ebay}prestashop>ebay_356c707afada9bf5305aee151b7e42c9'] = 'Diesen Shop zu meinen Favoriten hinzufügen';
-$_MODULE['<{ebay}prestashop>ebay_faeaec9eda6bc4c8cb6e1a9156a858be'] = 'Verfügbarkeit';
-$_MODULE['<{ebay}prestashop>ebay_8289b7ec22f39ace395a246dec26e79f'] = 'vorrätig';
+$_MODULE['<{ebay}prestashop>ebay_5096b1cd265224acf046bf2a04c97e98'] = 'Les valeur de l\'état de l\'objet défini dans votre configuration ne sont pas supportées par les catégories d\'eBay.';
+$_MODULE['<{ebay}prestashop>ebay_98dc9c312b853d330c69fbe11141e596'] = 'Vous pouvez modifier vos configuration d\'état de l\'objet dans l\'onglet paramètres. (Voir les états supportés par ebay : http://pages.ebay.fr/help/sell/item-condition.html)';
+$_MODULE['<{ebay}prestashop>ebay_1ec3f6e91783ec459fcdb51c89ddf225'] = 'Une version future du module vous permettra de spécifier les valeurs d\'état de l\'objet par catégories';
```
Comment (Quetzacoalt91) [inline de.php]: Sentences are in French instead of German

File: de.php [hunk 3]
```diff
@@ -113,6 +115,12 @@
 $_MODULE['<{ebay}prestashop>form_1326a43a80b8241e21c809f80b997845'] = 'Eröffnen Sie Ihren eBay-Shop';
 $_MODULE['<{ebay}prestashop>form_3d9742de1b9742bd8031ec3f8334da22'] = 'Verlinken Sie Ihr eBay-Konto mit dem eBay Add-on';
 $_MODULE['<{ebay}prestashop>form_c9e6616dc1bb37bf1ab3d56156288a8f'] = 'eBay Add-on konfigurieren';
+$_MODULE['<{ebay}prestashop>form_5c79609876d64e33ff348736ba605867'] = 'Ouvrez une Boutique eBay gratuitement';
+$_MODULE['<{ebay}prestashop>form_f7781230f034f74844124eb49c04e684'] = 'Lancez ou boostez votre activité sur eBay, en profitant de';
+$_MODULE['<{ebay}prestashop>form_1c1d90087dfad5f38c4135ac25197e6d'] = '90 jours* d\'abonnement gratuits ';
+$_MODULE['<{ebay}prestashop>form_de02798502e46104ef6d3627c34721af'] = 'à la Boutique A la Une.';
+$_MODULE['<{ebay}prestashop>form_6b2535c9030827712b8315f4b33afaad'] = 'http://portal.ebay.fr/ouvrezuneboutiquegratuitement';
+$_MODULE['<{ebay}prestashop>form_7d5a27f1b53dc519eac7bb288f664892'] = 'Plus d\'infos';
```
Comment (Quetzacoalt91) [inline de.php]: Same comment about the language

File: classes/EbayCountrySpec.php (hunk 1 above)
Comment (thoma202) [inline classes/EbayCountrySpec.php]: Addded a fallback on UK

File: de.php (hunk 2 above)
Comment (thoma202) [inline de.php]: Done

File: de.php (hunk 3 above)
Comment (thoma202) [inline de.php]: Translated

File: en.php [hunk 4]
```diff
@@ -76,17 +130,16 @@
...
 $_MODULE['<{ebay}prestashop>form_1abcda2246172b8a723da723ba3cb9e0'] = 'eBay Site';
 $_MODULE['<{ebay}prestashop>form_920dc5e73ec934da7215f65aa6fc642e'] = 'Prestashop Shop';
 $_MODULE['<{ebay}prestashop>form_4994a8ffeba4ac3140beb89e8d41f174'] = 'Language';
 $_MODULE['<{ebay}prestashop>form_3c54123fc64232cfcad3cedd49266ce5'] = 'Nb Current Listings';
 $_MODULE['<{ebay}prestashop>form_004bf6c9a40003140292e97330236c53'] = 'Action';
-$_MODULE['<{ebay}prestashop>form_3e9983cf1885a5ec9f5a5d8127137bd2'] = 'Delete Profile';
 $_MODULE['<{ebay}prestashop>form_0599b1ea4ecbec20a40188dca59e5ff9'] = 'The bold profile is your current profile. To change the profile you are currently working with, click on the desired profile';
 $_MODULE['<{ebay}prestashop>form_15f046413a3aed81622b21bf550add71'] = 'Add a New Profile';
 $_MODULE['<{ebay}prestashop>form_7fcf7ae95171549f39479f4983cef281'] = 'eBay Module Status';
+$_MODULE['<{ebay}prestashop>form_f260c8e81a413247fbdc053775e79046'] = 'Vous n\'avez pas encore défini de profil';
```
Comment (thoma202) [inline en.php]: This is only displayed for french, italian and spanish users

File: nl.php [hunk 5]
```diff
@@ -120,6 +115,12 @@
 $_MODULE['<{ebay}prestashop>form_1326a43a80b8241e21c809f80b997845'] = 'Open je eBay-winkel';
 $_MODULE['<{ebay}prestashop>form_3d9742de1b9742bd8031ec3f8334da22'] = 'Koppel je eBay-account aan de add-on van eBay';
 $_MODULE['<{ebay}prestashop>form_c9e6616dc1bb37bf1ab3d56156288a8f'] = 'Configureer de add-on van eBay';
+$_MODULE['<{ebay}prestashop>form_5c79609876d64e33ff348736ba605867'] = 'Ouvrez une Boutique eBay gratuitement';
+$_MODULE['<{ebay}prestashop>form_f7781230f034f74844124eb49c04e684'] = 'Lancez ou boostez votre activité sur eBay, en profitant de';
+$_MODULE['<{ebay}prestashop>form_1c1d90087dfad5f38c4135ac25197e6d'] = '90 jours* d\'abonnement gratuits ';
+$_MODULE['<{ebay}prestashop>form_de02798502e46104ef6d3627c34721af'] = 'à la Boutique A la Une.';
+$_MODULE['<{ebay}prestashop>form_6b2535c9030827712b8315f4b33afaad'] = 'http://portal.ebay.fr/ouvrezuneboutiquegratuitement';
+$_MODULE['<{ebay}prestashop>form_7d5a27f1b53dc519eac7bb288f664892'] = 'Plus d\'infos';
```
Comment (thoma202) [inline nl.php]: This is only displayed for french, italian and spanish users

File: pl.php [hunk 6]
```diff
@@ -120,6 +115,12 @@
 $_MODULE['<{ebay}prestashop>form_1326a43a80b8241e21c809f80b997845'] = 'Otwórz Sklep eBay';
 $_MODULE['<{ebay}prestashop>form_3d9742de1b9742bd8031ec3f8334da22'] = 'Powiąż swoje konto eBay z modułem eBay';
 $_MODULE['<{ebay}prestashop>form_c9e6616dc1bb37bf1ab3d56156288a8f'] = 'Skonfiguruj moduł eBay';
+$_MODULE['<{ebay}prestashop>form_5c79609876d64e33ff348736ba605867'] = 'Ouvrez une Boutique eBay gratuitement';
+$_MODULE['<{ebay}prestashop>form_f7781230f034f74844124eb49c04e684'] = 'Lancez ou boostez votre activité sur eBay, en profitant de';
+$_MODULE['<{ebay}prestashop>form_1c1d90087dfad5f38c4135ac25197e6d'] = '90 jours* d\'abonnement gratuits ';
+$_MODULE['<{ebay}prestashop>form_de02798502e46104ef6d3627c34721af'] = 'à la Boutique A la Une.';
+$_MODULE['<{ebay}prestashop>form_6b2535c9030827712b8315f4b33afaad'] = 'http://portal.ebay.fr/ouvrezuneboutiquegratuitement';
+$_MODULE['<{ebay}prestashop>form_7d5a27f1b53dc519eac7bb288f664892'] = 'Plus d\'infos';
```
Comment (thoma202) [inline pl.php]: This is only displayed for french, italian and spanish users

**Comment by Quetzacoalt91** [comment]
Hi @thoma202,

Thanks for all the changes, your PR has just been applied.

Best regards

**Pull request merged**